node-commons/src/main/java/org/apache/iotdb/commons/client/ClientManager.java

```diff
@@ -34,7 +34,7 @@
 
   private final KeyedObjectPool<K, V> pool;
 
-  public ClientManager(IClientPoolFactory<K, V> factory) {
+  protected ClientManager(IClientPoolFactory<K, V> factory) {
```

Comment:
`protected` means this method can be invoked by derived classes from other packages. Seems package-private is a better choice?

Reply:
Fixed